src/lib/machines/labware/labwareMachine.ts

```diff
@@ -424,10 +425,7 @@ export const createLabwareMachine = (
           ) {
             return;
           }
-          const matchResult = e.data.message.match(/^.*\s:\s(.*)$/);
-          if (matchResult && matchResult.length > 1) {
-            ctx.errorMessage = matchResult[1];
-          }
+          ctx.errorMessage = handleFindError(e.data);
```

Comment:
@BenTopping With this, server error extraction is now based on extractServerErrors method, right?

Reply:
Yep. HandleFindError calls extractServerErrors